lexicons/place/stream/default/metadata.json

```
@@ -0,0 +1,166 @@
+        "properties": {
+          "contentWarnings": {
+            "type": "array",
+            "items": {
+              "type": "string",
+              "knownValues": [
+                "place.stream.default.metadata#death",
+                "place.stream.default.metadata#drugUse",
+                "place.stream.default.metadata#fantasyViolence",
+                "place.stream.default.metadata#flashingLights",
+                "place.stream.default.metadata#language",
+                "place.stream.default.metadata#nudity",
+                "place.stream.default.metadata#PII",
+                "place.stream.default.metadata#sexuality",
+                "place.stream.default.metadata#suffering",
+                "place.stream.default.metadata#violence"
+              ]
+            },
+            "description": "Content warnings for this stream."
+          },
+          "contentRights": {
+            "type": "ref",
+            "ref": "#contentRights"
+          },
+          "distributionPolicy": {
+            "type": "ref",
+            "ref": "#distributionPolicy"
+          }
+        }
+      }
+    },
+    "contentRights": {
+      "type": "object",
+      "description": "Content rights and attribution information.",
+      "properties": {
+        "creator": {
+          "type": "string",
+          "description": "Name of the creator of the work."
+        },
+        "copyrightNotice": {
+          "type": "string",
+          "description": "Copyright notice for the work."
+        },
+        "copyrightYear": {
+          "type": "integer",
+          "description": "Year of creation or publication."
+        },
+        "license": {
+          "type": "string",
+          "knownValues": [
+            "https://creativecommons.org/licenses/by-nc-nd/4.0/",
+            "https://creativecommons.org/licenses/by-nc-sa/4.0/",
+            "https://creativecommons.org/licenses/by-nc/4.0/",
+            "https://creativecommons.org/licenses/by-nd/4.0/",
+            "https://creativecommons.org/licenses/by-sa/4.0/",
+            "https://creativecommons.org/licenses/by/4.0/",
+            "https://creativecommons.org/publicdomain/zero/1.0/",
+            "place.stream.default.metadata#all-rights-reserved"
+          ],
+          "description": "License URL or identifier."
+        },
+        "creditLine": {
+          "type": "string",
+          "description": "Credit line for the work."
+        }
+      }
+    },
+    "distributionPolicy": {
+      "type": "object",
+      "description": "Distribution and rebroadcast policy.",
+      "properties": {
+        "allowArchive": {
+          "type": "boolean",
+          "description": "Whether nodes can archive this stream."
+        },
+        "broadcastExpiry": {
+          "type": "string",
+          "format": "datetime",
+          "description": "When rebroadcast permissions expire (ISO 8601 format). If not specified, there is no expiration on rebroadcast permission."
```

Lexicon string formats are actually not exactly ISO 8601. See https://atproto.com/specs/lexicon#datetime
```suggestion
          "description": "When rebroadcast permissions expire. If not specified, there is no expiration on rebroadcast permission."
```